frontend/pages/admin/AppSettingsPage/AppSettingsPage.tsx

```diff
@@ -63,68 +98,174 @@ const AppSettingsPage = (): JSX.Element => {
-  const scrollInto = (elementId: string) => {
-    const yOffset = -215; // headers and tabs
-    const element = document.getElementById(elementId);
+  const showSection = (linkString: string) => {
+    setShowInfo(false);
+    setShowWebAddress(false);
+    setShowSso(false);
+    setShowSmtp(false);
+    setShowAgents(false);
+    setShowHostStatusWebhook(false);
+    setShowStatistics(false);
+    setShowAdvanced(false);
 
-    if (element) {
-      const top =
-        element.getBoundingClientRect().top + window.pageYOffset + yOffset;
-      window.scrollTo({ top });
+    switch (linkString) {
+      case "webaddress":
+        setShowWebAddress(true);
+        return;
+      case "sso":
+        setShowSso(true);
+        return;
+      case "smtp":
+        setShowSmtp(true);
+        return;
+      case "agents":
+        setShowAgents(true);
+        return;
+      case "host-status-webhook":
+        setShowHostStatusWebhook(true);
+        return;
+      case "statistics":
+        setShowStatistics(true);
+        return;
+      case "advanced":
+        setShowAdvanced(true);
+        return;
+      default:
+        setShowInfo(true);
     }
   };
 
+  useEffect(() => {
+    showSection(sectionTitle);
+  }, [sectionTitle]);
+
+  const renderSection = () => {
+    if (!isLoadingConfig && appConfig) {
+      return (
+        <>
+          {showInfo && (
+            <Info appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+          {showWebAddress && (
+            <WebAddress appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+          {showSso && <Sso appConfig={appConfig} handleSubmit={onFormSubmit} />}
+          {showSmtp && (
+            <Smtp appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+          {showAgents && (
+            <AgentOptions appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+          {showHostStatusWebhook && (
+            <HostStatusWebhook
+              appConfig={appConfig}
+              handleSubmit={onFormSubmit}
+            />
+          )}
+          {showStatistics && (
+            <Statistics appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+          {showAdvanced && (
+            <Advanced appConfig={appConfig} handleSubmit={onFormSubmit} />
+          )}
+        </>
+      );
+    }
+
+    return <></>;
+  };
+
   return (
     <div className={`${baseClass} body-wrap`}>
       <p className={`${baseClass}__page-description`}>
-        Set your organization information and configure SAML and SMTP.
+        Set your organization information and configure SSO and SMTP
       </p>
       <div className={`${baseClass}__settings-form`}>
         <nav>
           <ul className={`${baseClass}__form-nav-list`}>
             <li>
-              <a onClick={() => scrollInto("organization-info")}>
+              <Link
+                className={`${baseClass}__nav-link${
```

Comment:
@lukeheath if you can think of a better way to bold what's clicked, that would be amazing